src/Mailgun/Mailgun.php

```diff
@@ -166,6 +224,14 @@ public function setApiVersion($apiVersion)
     {
         $this->restClient->setApiVersion($apiVersion);
 
+        foreach ($this->regionClients as $regionClient) {
+            if ($regionClient == null) {
```

Comment:
This should be compared strictly `===` or using the PHP function `is_null()`.

Also, please consider to use Yoda conditions, specially when you compare using `==`. Example:
`if ($regionClient = null)` will return **always** true and this is a human error but `if (null = $regionClient)` will return an error and it means we will prevent an unexpected behaviour.